src/V3LinkParse.cpp

```diff
@@ -62,6 +62,7 @@ class LinkParseVisitor final : public VNVisitor {
     int m_genblkAbove = 0;  // Begin block number of if/case/for above
     int m_genblkNum = 0;  // Begin block number, 0=none seen
     VLifetime m_lifetime = VLifetime::STATIC;  // Propagating lifetime
+    bool m_topIfacesSupported = false;
```

Comment:
Comment all members please.

Reply:
N.A. anymore as I got rid of this member variable by implementing the below recommended change.